package.json

```diff
@@ -24,6 +24,7 @@
     "@mapbox/geojson-merge": "^1.1.1",
     "@mapbox/point-geometry": "^0.1.0",
     "@mapbox/togeojson": "^0.16.2",
+    "@maplibre/maplibre-gl-style-spec": "^23.3.0",
```

Comment:
We need this because of protomaps-leaflet upgrade, or we will use it directly in some places?